docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+User Guide
+=======
+SETA is a **desktop application for CS2103T Teaching Assistants** to track their students’ and tutorials’ details, and
+questions asked by students. SETA is optimized for use via a Command Line Interface (CLI) while still having the
+benefits of a Graphical User Interface (GUI). If you can type fast, SETA enables you to track your students, manage your
+tutorials and note down questions more effectively than traditional GUI apps.
+
+The purpose of this guide is to inform you what features are available in SETA, and how to use them to solve your 
+needs as a CS2103T TA.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: What is a Command Line Interface (CLI)?**<br>
+
+It is a programme that allows you to create, modify and delete files using a text-based user interface. In SETA, all 
+you have to do is to type in the command box and press enter to track your students and manage your questions and 
+tutorials.
+
+</div>
+
+* Table of Contents
+    * **[Quick Start](#quick-start)**
+    * **[Features](#features)**
+        * **[Student](#students)**
+            * Adding a student: [`addstu`](#adding-a-student--addstu)
+            * Editing a student: [`editstu`](#editing-a-student-editstu)
+            * Adding student's attendance: [`attendance`](#adding-students-attendance--attendance)
+            * Adding student’s response: [`addresponse`](#adding-students-response-addresponse)
+            * Adding help tag to a student: [`helpstu`](#adding-help-tag-helpstu)
+            * Deleting a student: [`deletestu`](#deleting-a-student-deletestu)
+            * Finding a student: [`findstu`](#finding-a-student-findstu)
+            * Listing all students: [`liststu`](#listing-all-students-liststu)
+        * **[Questions](#questions)**
+            * Adding a question: [`addq`](#adding-a-question--addq)
+            * Marking a question: [`markq`](#marking-a-question--markq)
+            * Unmarking a question: [`unmarkq`](#unmarking-a-question--unmarkq)
+            * Deleting a question: [`deleteq`](#deleting-a-question--deleteq)
+        * **[Tutorials](#tutorials)**
+            * Adding a tutorial: [`addtut`](#adding-a-tutorial--addtut)
+            * Deleting a tutorial: [`deletetut`](#deleting-a-tutorial--deletetut)
+            * Marking a tutorial: [`marktut`](#marking-a-tutorial-marktut)
+        * **Exiting the program**: [`exit`](#exiting-the-program--exit)
+    * **[FAQ](#faq)**
+    * **[Command Summary](#command-summary)**
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Quick Start
+
+1. Ensure you have Java `11` or above installed in your Computer.
```

Comment:
to add hyperlink to install Java 11, and we can add a link to help page from Java website for users facing trouble with installing Java 11?

Reply:
added